Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>I hit an error in this cell where the default date range resulted in no incidents showing.</p><p>Thinking it might be better to have this kind of logic:</p><ul><li>Get the incidents</li><li>Calculate start and end dates from the incident dates.</li><li>Show the timeline for all retrieved incidents</li><li>Then show the date picker to select the date range to display in the selection list below.</li></ul><p> </p><p>Also:</p><ul><li>probably worth catching the case where no incidents are returned - instead of letting the timeline error out.</li><li>Use the mp_plot.timeline pandas accessor - I'm thinking it would be good to push people towards invoking things through this/other accessors or pivot funcs</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>